Src/ILGPU/Runtime/OpenCL/CLAccelerator.cs

```diff
@@ -527,12 +527,25 @@ protected override CLKernel CreateKernel(CLCompiledKernel compiledKernel)
             new CLKernel(this, compiledKernel, launcher);
 
         /// <summary cref="Accelerator.CreateStream"/>
-        protected override AcceleratorStream CreateStreamInternal() =>
-            new CLStream(this);
+        protected override AcceleratorStream CreateStreamInternal()
+        {
+            var stream = new CLStream(this);
+            ChildObjects.Add(new WeakReference<AcceleratorObject>(stream));
```

Comment:
ISSUE: You should not add this reference to the child list since it will be added automatically to the list in its own constructor.

Reply:
When I created a stream, it didn't add itself to the list. I compared pointers, and the only stream in the list was the default stream. Is this potentially another bug?

Reply:
Well, whatever I did, I must have done it wrong. They're in the list now...